frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -593,16 +592,35 @@ export const dashboardLogic = kea<dashboardLogicType>([
             () => [router.selectors.searchParams],
             (searchParams) => searchParams.highlightInsightId,
         ],
-        lastRefreshed: [
+        sortedDates: [
             (s) => [s.insightTiles],
-            (insightTiles): Dayjs | null => {
+            (insightTiles): Dayjs[] => {
                 if (!insightTiles || !insightTiles.length) {
-                    return null
+                    return []
                 }
 
                 const validDates = insightTiles.map((i) => dayjs(i.last_refresh)).filter((date) => date.isValid())
-                const oldest = sortDayJsDates(validDates)
-                return oldest.length > 0 ? oldest[0] : null
+                return sortDayJsDates(validDates)
+            },
+        ],
+        lastRefreshed: [
```

Comment:
I think for clarity we should rename this to `newestRefreshed`. Two benefits:
1. Clearly analogous to `oldestRefreshed`
2. Maybe even more importantly, it's in fact not possible when the dashboard was refreshed, because that's not how querying works at all – it's _all_ based on individual insights. For example, let's say you have a dashboard with two items and both have results from 3 hours ago. The dashboard has no overrides or filters. Then you create a new insight, save it, and add it to the dashboard. The new insight will have results from 5 minutes ago in the cache, so `newestRefreshed` is suddenly 5 minutes ago, despite the dashboard never being refreshed as a whole.